src/plugins/intel_cpu/tests/functional/subgraph_tests/src/subgraph_serialize.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/openvino.hpp"
+#include "openvino/opsets/opset9.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/convolution_params.hpp"
+#include "snippets/op/subgraph.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::opset9;
+
+namespace SubgraphTestsDefinitions {
+
+class SubgraphSnippetSerializationTest : public ::testing::Test, public CPUTestsBase {};
+
+TEST_F(SubgraphSnippetSerializationTest, SerializeSubgraph) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    auto model = ([] () -> std::shared_ptr<ov::Model> {
+        auto shape = ov::Shape({2, 2});
+        auto input0 = std::make_shared<Parameter>(ov::element::f32, shape);
+        auto input1 = std::make_shared<Parameter>(ov::element::f32, shape);
+        auto ininput0 = std::make_shared<Parameter>(ov::element::f32, shape);
+        auto ininput1 = std::make_shared<Parameter>(ov::element::f32, shape);
+        auto add = std::make_shared<Add>(ininput0, ininput1);
+        auto subgraph_body = std::make_shared<ov::Model>(ov::NodeVector{add}, ov::ParameterVector{ininput0, ininput1});
+        auto subgraph = std::make_shared<ngraph::snippets::op::Subgraph>(ov::NodeVector{input0, input1}, ov::clone_model(*subgraph_body.get()));
+        return std::make_shared<ov::Model>(ov::NodeVector{subgraph}, ov::ParameterVector{input0, input1});
+    })();
+    ov::Core core;
+    ov::CompiledModel compiled_model = core.compile_model(model, "CPU");
+    std::stringstream stream;
+    compiled_model.export_model(stream);
+    ov::CompiledModel imported_compiled_model = core.import_model(stream, "CPU");
+    float data[] = {1.f, 1.f, 1.f, 1.f};
+    ov::Tensor input_data1{ov::element::f32, ov::Shape({2, 2}), data};
+    ov::Tensor input_data2{ov::element::f32, ov::Shape({2, 2}), data};
+    ov::InferRequest infer_request = compiled_model.create_infer_request();
+    infer_request.set_input_tensor(0, input_data1);
```

Comment:
If you inherit from SnippetsTestsCommon or from ov::test::SubgraphBase, like the snippets tests do, you will not have to fill inputs or create infer_request manually. This will also allow you to minimize code duplication. Presently there is a lot of code duplication between `SerializeSubgraph` and `SerializeSubgraphWithScalarConst`.